DocumentedExamples/Meridional_heat_transport.ipynb

```diff
@@ -5,15 +5,19 @@
    "metadata": {},
```

Comment:
<p>Re "for this method to work" -&gt; You're referring here to the potential dependence of the meridional heat transport on the reference temperature associated with the fact that there could be a net meridional volume flux. In fact, both the above methods (the <code>temp_flux_adv_int_z</code> and <code>net_sfc_heating</code>) also do not account for net meridional volume transports. Thus their MHT will depend on the reference temperature. </p><p> </p><p>Hence, for this calculation here I would suggest first including a version which just calculates the net meridional heat transport without worrying about the reference temperature dependence. This is the only version of this calculation that can be properly compared to methods 1 and 2. Also, for this method you should not need to use <code>dxt</code> or <code>dzt</code> , only the transport diagnostic itself.</p><p> </p><p>If you wanted to, you could also include a method 4 then which is the current method 3 -&gt; i.e. what is the difference if you do subtract off the reference-temperature-dependent component of MHT associated with the net meridional volume transport? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/285/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
I see your point. In that case, I think it might be useful to rewrite method 3 to be consistent with other methods, and then add a note that all of these methods are dependent on the temperature scale as there might be net meridional volume flux?

Reply:
<p>This comment still hasn't been addressed. I'd suggest just changing the wording at this point. In particular, you don't need the "For this method to work... southward flow". If you want to just compare to method 1, using <code>temp_flux_adv_int_z</code>,then you don't want to do this subtraction).</p><p> </p><p>I'd suggest also adding "mass" in between "meridional" and "transport" in the first sentence.</p><p> </p><p>EDIT: I see you have this extra text below. So I think you just missed removing it from here.</p>